src/modules/70-pipeline/components/PipelineStudio/FailureStrategy/FailureStrategy.tsx

```diff
@@ -83,24 +83,32 @@ export function FailureStrategy(props: FailureStrategyProps, ref: StepCommandsRe
     }
   }))
 
+  useEffect(() => {
+    debouncedUpdate({ failureStrategies: getInitialValues() })
+  }, [])
```

Comment:
Codacy has a fix for the issue: [React Hook useEffect has missing dependencies: 'debouncedUpdate' and 'getInitialValues'. Either include them or remove the dependency array.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232232)
```suggestion
  }, [debouncedUpdate, getInitialValues])
```